modules/gateway.go

```diff
@@ -40,11 +25,33 @@ type NetConn interface {
 // RPCFunc is the type signature of functions that handle incoming RPCs.
 type RPCFunc func(NetConn) error
 
+// A NetAddress contains the information needed to contact a peer.
+type NetAddress string
+
 type GatewayInfo struct {
 	Address NetAddress
 	Peers   []NetAddress
 }
 
+// TODO: Move this and it's functionality into the gateway package.
+var (
+	BootstrapPeers = []NetAddress{
+		"23.239.14.98:9988",
+	}
+)
```

Comment:
my rationale is that it should be up to the gateway to decide how & where the bootstrapping happens. It's whole job is to find other peers and stay synchronized, that should include the bootstrap peers. We don't have siad tell consensus what the genesis block is.